src/components/Button.astro

```diff
@@ -0,0 +1,32 @@
+---
+interface Props {
+    variant?: keyof typeof Variant;
+    size?: keyof typeof Size;
+    icon?: string;
+    disabled?: boolean;
+    label: string;
+    onClick?: () => void;
+}
+
+const style = 'inline-flex items-center justify-center rounded-md border disabled:cursor-not-allowed disabled:bg-primary-100 disabled:text-primary-400';
+
+const Variant = {
+    primary: 'bg-primary-600 text-[#fff] hover:bg-primary-800 active:bg-primary-950 border-primary-600 hover:border-primary-800 active:border-primary-950',
+    secondary: 'text-primary-600 hover:text-primary-800 hover:bg-[#F4F4F5] active:text-primary-950',
+    filled: 'bg-white text-white text-primary-600 hover:text-primary-800 hover:bg-[#F4F4F5] active:text-primary-950',
```

Comment:
The #F4F4F5 is not in our colors?

Reply:
In figma it is a neutral color, but it is not in the tailwind file yet, only the primary and secondary, I can change it to other secondary